build/DropFiles.targets

```diff
@@ -3,7 +3,6 @@
   <ItemGroup>
     <FilesToSign Include="@(DropSignedFile)">
       <Authenticode>Microsoft400</Authenticode>
-      <StrongName>StrongName</StrongName>
```

Comment:
><StrongName>StrongName</StrongName> [](start = 6, length = 35)

Is this set somewhere else?

Reply:
Good catch, I was doing this to test release builds.

Reply:
I think I need to Condition this  if SignType == 'real'.

Reply:
Is this the right condition, or should it really be lab configuration based? I thought you needed this for test signing support.

Reply:
Oh, I see. Nevermind.